packages/gitbook/src/components/Search/SearchResults.tsx

```diff
@@ -175,7 +175,7 @@ export const SearchResults = React.forwardRef(function SearchResults(
         [moveBy, select],
     );
 
-    if (!results) {
+    if (!results || results.filter((result) => result.type !== 'question').length === 0) {
```

Comment:
```suggestion
    if (!results?.some((result) => result.type !== 'question')) {
```

.some exit earlier than .filter

Reply:
Updated that logic and added the changeset